modules/AWS/eks/main.tf

```diff
@@ -25,11 +27,12 @@ module "eks" {
 
   node_groups = {
     appNodes = {
-      name             = "appNode"
+      name             = "appNode-${replace(join("-", var.instance_types), ".", "_")}"
```

Comment:
Is this to make it more descriptive or required for auto scaling?

Reply:
This is to make it more descriptive but also enable the change of the instance_type. I haven't tested that thoroughly but it will get rid of the static `appNode` string. When you change the instance type, a new node group is created and after it is created, the workload is migrated there and the previous node group is destroyed.